examples/jogging.py

```diff
@@ -0,0 +1,129 @@
+"""Example of moving a motor using the supervisor."""
+
+import argparse
+import time
+import os
+import sys
+
+sys.path.append(os.path.join("actuator"))
```

Comment:
oh, you shouldn't need this - if you do `pip install -e '.[dev]'` from from directory with the setup.py file, it should add actuator package to your python path

also can run with `python -m examples.jogging` instead of having to run `python examples/jogging.py`
```suggestion
```